mobile/src/hooks/getHealthKitData.tsx

```diff
@@ -1,39 +1,39 @@
-import {useState} from 'react';
+import { useState } from 'react';
 import AppleHealthKit from 'rn-apple-healthkit';
 
-const PERMS = AppleHealthKit.Constants.Permissions;
+// const PERMS = AppleHealthKit.Constants.Permissions;
 
 const getHealthKitData = () => {
   const [data, setData] = useState({
-    gender: null
+    gender: null,
   });
 
-  const updateData = (field,value) => {
+  const updateData = (field, value) => {
       setData({
         ...data,
-        [field]: value
+        [field]: value,
       });
   };
 
-  const options = {
-    permissions: {
-      read: [PERMS.BiologicalSex],
-    },
-  };
-
-  AppleHealthKit.initHealthKit(options, (err, results) => {
-    if (err) {
-        console.log("error initializing Healthkit: ", err);
-        return;
-    }
-});
-
-  AppleHealthKit.getBiologicalSex(null, (err, results) => {
-    if (err) {
-      console.log(`Error returning sex`, err);
-    } 
-    updateData("gender",results);
-});
+  // const options = {
+  //   permissions: {
+  //     read: [PERMS.BiologicalSex],
+  //   },
+  // };
+
+//   AppleHealthKit.initHealthKit(options, (err, results) => {
+//     if (err) {
+//         console.log('error initializing Healthkit: ', err);
+//         return;
+//     }
+// });
+
+//   AppleHealthKit.getBiologicalSex(null, (err, results) => {
+//     if (err) {
+//       console.log(`Error returning sex`, err);
+//     }
+//     updateData('gender', results);
+// });
```

Comment:
What's going on here? Should we try to get it to work rather than commenting out?

Reply:
I think it was throwing TSC errors and I knew @SukiCai was working on this so I just commented it out. Probably best to revert to master state